### ISSUE TITLE
Compatibility with radiora2

### ISSUE THREAD

**Issue by jfinestone** [rnilssoncx/homebridge-pico#43]
Is this compatible with radiora2? Or would it be possible to make it compatible with radiora2?. Thank you.

**Comment by scoutbeer** [comment]
I think this plug-in relies on the telnet integration/access.  RadioRA2 allows for that.  So i'm hoping that @rnilssoncx can confirm.

I was also wondering if the 5/10/15 button seeTouch tabletop RF keypads will work with this plugin?

**Comment by jfinestone** [comment]
You may want to look at the other radiora2 plugin.  The table top keypads
support feedback whereas the PICOs don’t. And I don’t believe that this
plug-in supports feedback/status so the other plug-in may be better suited
for your application.   Notwithstanding that, this is an excellent plug-in
and works well.

**Comment by scoutbeer** [comment]
Thanks for the reply didn't even know that RA2 plugin existed!  I'm guessing you are referring to this one? https://github.com/senorshaun/homebridge-radiora2 
I'll probably end up using both as this plugin has the exact functions i'm looking for but was hoping to also use a 10-button seeTouch.

**Comment by jfinestone** [comment]
Yes. That’s it. But neither of these are my plug-in. Not sure why I got
copied on this message. I just use both of them.

The pico one is especially interesting because it allows you to create a
automations in HomeKit as a result of a pico press. So for example a price
of a pico will control whatever you have it controlling in Lutron but could
also controlling on Lutron device.

As you’ll see in the directions there are several different options for
single press double press press and hold etc. Keep in mind that only a
single press is supported in Lutron, but this plug-in will allow you to do
other things — whatever you can do in HomeKit — with the alternate
interactions.
